ROADMAP.md

```diff
@@ -153,28 +153,45 @@ Any dates listed below and the specific issues that will ship in a given milesto
-* Host-aware Stack Manager [#1038](https://github.com/crossplane/crossplane/issues/1038)
-  * Enables deploying multiple Crossplane instances watching different Kubernetes API servers (i.e. multiple API servers running in different namespaces) on a single Host Kubernetes cluster.
-* Designs for:
-   * Defining your own claim kinds [#1106](https://github.com/crossplane/crossplane/issues/1106) 
-   * Allowing a claim to be satisfied by multiple resources [#1105](https://github.com/crossplane/crossplane/issues/1105)
-   * Versioning and upgrade support [#879](https://github.com/crossplane/crossplane/issues/879), [#435](https://github.com/crossplane/crossplane/issues/435)
-* GCP storage buckets to v1beta1 [crossplane/stack-gcp#130](https://github.com/crossplane/stack-gcp/issues/130)
-* GCP networking resources to v1beta1 [crossplane/stack-gcp#131](https://github.com/crossplane/stack-gcp/issues/131)
-* Improved logging and eventing [crossplane/crossplane-runtime#104](https://github.com/crossplane/crossplane-runtime/issues/104)
+## [v0.8.0 Stacks simplify cloud-native app and infrastructure provisioning](https://github.com/crossplane/crossplane/releases/tag/v0.8.0)
+- Stacks for ready-to-run cloud environments (GCP, AWS, Azure) [#1136](https://github.com/crossplane/crossplane/issues/1136)
+  - Spin up secure cloud environments (nets, subnets, secure service connectivity, k8s clusters, resource classes, etc.) with just a few lines of yaml (single CR)
+- Support for PostgreSQL 11 on the `PostgreSQLInstance` claim (thanks first-time contributor @vasartori!) [#1245](https://github.com/crossplane/crossplane/pull#1245)
+- RBAC group and role refinements
+- Enable [stack-aws](https://github.com/crossplane/stack-aws) to authenticate to the AWS API using [IAM Roles for Service Accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) when running on EKS [stack-aws#126](https://github.com/crossplane/stack-aws/pull/126)
+- Improved logging and eventing [crossplane/crossplane-runtime#104](https://github.com/crossplane/crossplane-runtime/issues/104)
+- [Observability guide](https://github.com/crossplane/crossplane/blob/master/docs/observability-developer-guide.md) for logging and eventing in Crossplane controllers
+- Support default select values in the UI schema for Crossplane resources
+- Host-aware Stack Manager [#1038](https://github.com/crossplane/crossplane/issues/1038)
+  - Enables deploying multiple Crossplane instances watching different Kubernetes API servers on a single Host Kubernetes cluster.
+- Template Stacks (experimental)
+  - Helm and kustomize engine support for pluggable rendering
+  - Ported [stack-minimal-gcp](https://github.com/crossplane/stack-minimal-gcp/pull/1) and [sample-stack-wordpress](https://github.com/crossplane/sample-stack-wordpress/pull/31) to use Template Stacks and published to https://hub.docker.com/u/crossplane
+
+## v0.9.0
```

Comment:
other v0.9 items:

* Stacks Manager support for private repos and robot account credentials
* Release process and efficiency improvements

Reply:
done